.github/workflows/check_liveness_after_site_deployment.yml

```diff
@@ -0,0 +1,35 @@
+# .github/workflows/test01.yaml
+name: check_liveness_after_site_deployment.yml
+
+env:
+   AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY_ID}}
+   AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY}}
+   AWS_REGION: ${{secrets.AWS_REGION}}
+   
+   API_KEY_TEST_MAIL: ${{secrets.API_KEY_TEST_MAIL}}
+   TEST_MAIL_NAMESPACE: ${{secrets.TEST_MAIL_NAMESPACE}}
+   
+on:
+  push:
+    paths:
+      - '**.tf'
+      - 'html/**'
+      - 'api/**'
+      - 'infra/templates/**' 
```

Comment:
The workflow should be executed after the code is deployed, so you need to add it after the check readiness in deploy_push.yml after check readiness https://github.com/mongulu-cm/mtchoun-mouh/blob/40bf4625e79652b750c1a8daf87b8874fe6ea58c/.github/workflows/deploy_push.yml#LL99-L105C24  or use your call your workflow from there. 

I think it's possible for deduplication https://github.com/mongulu-cm/lobembe/issues/32 but don't know if it's possible to call workflow located in nother file. It's up you to choose one of the solutions.